### PR TITLE
Release/0.4.0

### PR DESCRIPTION
**Pull request by jbeemster**

### CODE REVIEW

File: snowplowtrk.go [hunk 1]
```diff
@@ -16,18 +16,19 @@ package main
 import (
 	"encoding/json"
 	"errors"
+	gt "github.com/snowplow/snowplow-golang-tracker/v2/tracker"
```
Comment (paulboocock) [inline snowplowtrk.go:19]: Seeing this in action makes me happy!

**Comment by jbeemster** [comment]
That should all be addressed now @paulboocock !

> Side note: Writing context arrays in terminal is hard work

Yeah really more aimed at shell scripts at any level of complexity...

**Comment by paulboocock** [comment]
LGTM

**Comment by alexanderdean** [comment]
> Yeah really more aimed at shell scripts at any level of complexity...

Does something like `jq` still help here?

**Comment by jbeemster** [comment]
I have generally used `jq` to parser values _out_ of JSON and less to create JSON encoded strings.  Quick look at the docs does suggest it is possible however!

Another option would be to allow different more bash friendly input modes like some sort of delimiter separated list of KV pairs - we can likely take some inspiration from the aws-cli here as they offer both JSON and alternative input modes.

**Pull request merged**